planning/behavior_path_avoidance_module/src/debug.cpp

```diff
@@ -188,10 +188,25 @@ MarkerArray createObjectInfoMarkerArray(const ObjectDataArray & objects, std::st
   return msg;
 }
 
+MarkerArray createOverhangLaneletMarkerArray(const ObjectDataArray & objects, std::string && ns)
```

Comment:
Will there be additional documentation or image showing the usage of this marker?

Reply:
@Owen-Liuyuxuan Thank you for your comment.

Unfortunatelly, there is no document about overhang lanelet. It is one of the ego driving lanelets (closest lanelet to the object) and used in the logic to check whether the object is on the ego lane.

Actually, it is difficult to understand without any comment. So, I'll add above brief explanation on code.

Reply:
@Owen-Liuyuxuan I updated comment in code in https://github.com/autowarefoundation/autoware.universe/pull/6601/commits/1344e6c9295eaf6feffe0dc6b83659d5435a2705.